### PR TITLE
matt-gabe

### PR DESCRIPTION
**Pull request by famavott**
#### Single-line Summary
**Today, Matt and Gabe paired together. It took about 1.5 hours**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we added the CDN for handlebars to the HTML page.
  2. Next, we set up the handlebars template in the head of the HTML file.
  3. We compiled our template with handlebars to prepare to send our object data to our template in HTML.
 4. Constructed an object literal, and then rendered HTML with the handlebar template function. 

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] I have entered the Pull Request title in the format of "firstname-lastname"
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: matt-gabe/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <title>Lab 4</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="styles/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <script id="article-template" type="text/x-handlebars-template">
+      <article class="template" data-category="{{ category }}">
```
Comment (izzybaer) [inline matt-gabe/index.html:14]: the author and category filters are broken, the solution will be here inside of the template

File: matt-gabe/scripts/article.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  var actualTemplate = Handlebars.compile(sourceHtml);
+
+  // REVIEW: If your template will use properties that aren't on the object yet, add them.
+  //   Since your template can't hold any JS logic, we need to execute the logic here.
+  //   The result is added to the object as a new property, which can then be referenced by key in the template.
+  //   For example, you might want to display how old a post is, or say "(draft)" if it has no publication date:
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  this.publishStatus = this.publishedOn ? `published ${this.daysAgo} days ago` : '(draft)'; // this is a "ternary operator"
+
+  // DONE: Use the function that Handlebars gave you to return your filled-in html template for THIS article.
```
Comment (izzybaer) [inline matt-gabe/scripts/article.js:28]: this is unnecessary, you just need the return statement `return actualTemplate`

Comment (gabrielx52) [reply matt-gabe/scripts/article.js:28]: That was from the starter code. We weren't sure what to do with it.

Comment (izzybaer) [reply matt-gabe/scripts/article.js:28]: sorry for the confusion, I was talking about lines 29-37.

**Pull request merged**